mod/admin.php

```diff
@@ -936,6 +936,10 @@ function admin_page_summary(App $a)
 		$showwarning = true;
 		$warningtext[] = L10n::t('Friendica\'s configuration now is stored in config/local.config.php, please copy config/local-sample.config.php and move your config from <code>.htconfig.php</code>. See <a href="%s">the Config help page</a> for help with the transition.', $a->getBaseURL() . '/help/Config');
 	}
+	if (file_exists('local.ini.php')) {
```

Comment:
Shouldn't you check in the config folder for that file?

Reply:
Only if the warning should be shown in a particular case *g*